web/thesauruses/rust/1/strings.json

```diff
@@ -0,0 +1,200 @@
+{
+  "meta": {
+    "language": "rust",
+    "language_version": "1",
+    "language_name": "Rust",
+    "structure": "strings"
+  },
+  "concepts": {
+    "is_primitive_or_not": {
+      "code": "Yes"
+    },
+    "import": {
+      "not-implemented": true
+    },
+    "default_string_byte_encoding": {
+      "code": "UTF-8"
+    },
+    "create_new_string": {
+      "code": "let foo = \"string\";"
+    },
+    "create_multiline_string": {
+      "code": "let foo = \"line 1 \\\nline 2\";"
+    },
+    "assign_new_string": {
+      "code": "let a = \"a\";\nlet b = a;"
+    },
+    "destroy_string": {
+      "code": "let foo = \"x\";\ndrop(foo);"
+    },
+    "length_of_string": {
+      "code": "let foo = \"stringText\".len();"
+    },
+    "max_length_of_string": {
+      "not-implemented": true
+    },
+    "clear_string": {
+      "code": "let mut foo = \"bar\";\nfoo = \"\";"
+    },
+    "is_empty": {
+      "code": "foo.is_empty()"
+    },
+    "concatenate_two_strings": {
+      "code": "let foo = \"foo\";\nlet bar = \"bar\";\nlet foobar = foo.to_owned() + bar;"
```

Comment:
[NB] I'm only vaguely familiar with Rust and had to look up what to_owned() was. Interesting!